src/tests/nativeaot/SmokeTests/Determinism/Determinism.cs

```diff
@@ -3,30 +3,45 @@
 
 using System;
 using System.IO;
+using Xunit;
 
-using var baseline = File.OpenRead("baseline.object");
-using var compare = File.OpenRead("compare.object");
+public class Program
+{
+    [Fact]
+    public static int TestEntryPoint()
+    {
+        var baseline = File.OpenRead("baseline.object");
+        var compare = File.OpenRead("compare.object");
 
-Console.WriteLine($"Baseline size: {baseline.Length}");
-Console.WriteLine($"Compare size: {compare.Length}");
+        Console.WriteLine($"Baseline size: {baseline.Length}");
+        Console.WriteLine($"Compare size: {compare.Length}");
 
-if (baseline.Length != compare.Length)
-    throw new Exception("Different sizes");
+        if (baseline.Length != compare.Length)
+        {
+            Console.WriteLine("Test Fail: Baseline and Compare have different sizes.");
+            return 101;
+        }
 
-long length = baseline.Length;
-for (int i = 0; i < length; i++)
-{
-    if (baseline.ReadByte() != compare.ReadByte())
-        throw new Exception($"Different at byte {i}");
-}
+        long length = baseline.Length;
+        for (int i = 0; i < length; i++)
+        {
+            if (baseline.ReadByte() != compare.ReadByte())
+            {
+                Console.WriteLine($"Test Fail: Baseline and Compare were different"
+                                  + " at byte {i}.");
+                return 101;
+            }
+        }
 
-// We're not interested in running this, we just want some junk to compile
-if (Environment.GetEnvironmentVariable("Never") == "Ever")
-{
-    Delegates.Run();
-    Devirtualization.Run();
-    Generics.Run();
-    Interfaces.Run();
-}
+        // We're not interested in running this, we just want some junk to compile
+        if (Environment.GetEnvironmentVariable("Never") == "Ever")
+        {
+            Delegates.Run();
+            Devirtualization.Run();
+            // Generics.Run();
```

Comment:
Seems like a functional change.

Reply:
Oh if you mean having `Generics.Run()` commented out, that's just temporarily. I was running into some build issues but I wanted to test in CI for a more complete feedback. It will be restored before moving this PR from "Draft" to "Ready for Review".